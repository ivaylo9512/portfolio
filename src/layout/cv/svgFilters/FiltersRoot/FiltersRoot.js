import ShapeShadow from '@cv/svgFilters/ShapeShadow';
import BlackShadow from '@cv/svgFilters/BlackShadow';
import NameShadow from '@cv/svgFilters/NameShadow';
import GithubShadow from '@cv/svgFilters/GithubShadow';
import ProjectShadow from '@cv/svgFilters/ProjectShadow';
import CircleShadow from '@cv/svgFilters/CircleShadow';
import DotShadow from '@cv/svgFilters/DotShadow';
import './FiltersRoot.scss'

const FiltersRoot = () => {
    return(
        <svg className='svg-root' xmlns='http://www.w3.org/2000/svg' xmlnsXlink='http://www.w3.org/1999/xlink'>
            <defs>
                <ShapeShadow />
                <CircleShadow />
                <DotShadow />
                <BlackShadow />
                <NameShadow />
                <ProjectShadow />
                <GithubShadow />
            </defs>
        </svg>
    )
}

export default FiltersRoot