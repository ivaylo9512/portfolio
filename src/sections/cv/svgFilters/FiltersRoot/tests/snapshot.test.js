import FiltersRoot from '../FiltersRoot';
import { shallow } from 'enzyme';

describe('FiltersRoot snapshot tests', () => {
    const createWrapper = () => shallow(
        <FiltersRoot />
    )

    it('should match snapshot', () => {
        const wrapper = createWrapper();
     
        expect(wrapper).toMatchSnapshot();
    })
});