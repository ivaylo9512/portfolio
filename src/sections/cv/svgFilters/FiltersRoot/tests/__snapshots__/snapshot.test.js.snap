// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FiltersRoot snapshot tests should match snapshot 1`] = `
<svg
  className="svg-root"
  xmlns="http://www.w3.org/2000/svg"
  xmlnsXlink="http://www.w3.org/1999/xlink"
>
  <defs>
    <ShapeShadow />
    <CircleShadow />
    <DotShadow />
    <RectangleWhite />
    <NameShadow />
    <ShapeShadow />
  </defs>
</svg>
`;
